Display.h

```diff
@@ -37,12 +37,14 @@
 
 #include "Utilities.h"
 
+#include "Indicators.h"
 
 namespace display {
 
 static const int kWidth = 20;  // Width of the display
 static const int kHeight = 4;  // Height of the display
-
+static const int kHeaderLength = 19;  // Desired header length
```

Comment:
Why desired?

Reply:
Last column of the first row (row=0) is reserved to an indicator (breathing assist indicator). The remaining 19 columns are available for text message